utils/types.go

```diff
@@ -135,6 +140,15 @@ func GetKongClient(opt KongClientConfig) (*kong.Client, error) {
 		tlsConfig.RootCAs = certPool
 	}
 
+	if opt.TLSClientCert != "" && opt.TLSClientKey != "" {
+		// Read the key pair to create certificate
+		cert, err := tls.LoadX509KeyPair(opt.TLSClientCert, opt.TLSClientKey)
```

Comment:
The variables contain the cert and key themselves while the function expects path to files that contain the necessary bits.

Reply:
Right my bad, will fix asap. Thanks.